recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
 {% set name = "meshpy" %}
 {% set uname = "MeshPy" %}
-{% set version = "2022.1.3" %}
-{% set sha256 = "a7158e31ece25fa6c6cebce9fd1e968157d661dc8769fb30ceba69c351478475" %}
+{% set version = "2025.1.1" %}
+{% set sha256 = "70fc707fe9ccd9e907b95a9271804b4dd02e77d60644f64a0384cbf9e6d5b86b" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ uname }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ uname[0] }}/{{ uname }}/{{ uname }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ uname[0] }}/{{ uname }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 4
+  number: 0
```

Comment:
To build only Python 3.10+, add this item here:
```yaml
  skip: true  # [py<310]
```

Reply:
Thanks! It's been a while, I forgot how to do this right.